plugins/statFunctions/questionCodeHelper.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * This file is part of statFunctions plugin
+ */
+namespace statFunctions;
+
+use Yii;
+use Survey;
+use SurveyDynamic;
+use CDbCriteria;
+
+class questionCodeHelper
+{
+    /** @var integer $surveyId **/
+    public $surveyId = 0;
+
+    /**
+     * @param integer $surveyId
+     */
+    public function __construct($surveyId)
+    {
+        $this->surveyId = $surveyId;
+        /* Throw error if surveyid is invalid ? */
+    }
+
+    /**
+     * Check the survey
+     * @param string $qCode question SGQA
```

Comment:
Allow EM code is on todo, but need really more work :) (and I think a EM code to SGQA must come from Question::model or theme …)